etc/kayobe/ansible/rabbitmq-reset.yml

```diff
@@ -68,4 +68,4 @@
     # The following services use RabbitMQ.
     - name: Restart OpenStack services
       shell: >-
-        systemctl -a | egrep '(barbican|blazar|cinder|cloudkitty|designate|heat|ironic|keystone|magnum|manila|neutron|nova|octavia)' | awk '{ print $1 }' | xargs systemctl restart
+        systemctl list-units --type=service | egrep '(barbican|blazar|cinder|cloudkitty|designate|heat|ironic|keystone|magnum|manila|neutron|nova|octavia)' | egrep -v ceph | awk '{ print $NF }' | xargs systemctl restart
```

Comment:
What about using a grep for `^kolla-'` instead of filtering out `ceph`?

Reply:
Something like `egrep 'kolla-(barbican|…)'`

Reply:
Done